### PR TITLE
Create Setting Up Python on IIS 7.5

### PR DESCRIPTION
**Pull request by BadCrumpany**
Migrating articles from the community to the knowledge center.

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a5d8a4e411/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-428d55de1e/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3b945dca8b/how-to/)

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by cyrichardson** [comment]
@RackerKate Will start tomorrow off with this one.

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-3f92ac7471/how-to/). :bow:

File: content/cloud-servers/set-up-python-on-iis-75.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+product_url: cloud-servers
+---
+
+Use the following steps to install and set up Python on Internet Information Services (IIS) 7.5:
+
+1. Ensure that IIS Common Gateway Interface (CGI) is installed through role services.
+2. Launch IIS Manager by selecting **Control Panel > Administrative Tools > Internet Information Services (IIS) Manager**.
+3. Click on your website and double-click on **Handler Mappings** in the center panel. 
+4. Click **Add Script Map** in the **Actions** box to the right.
+5. In the **Add Script Map** window, enter ***.py** as the **Request Path**, and **python.exe** as the **Executable**.
```
Comment (chri2547) [inline content/cloud-servers/set-up-python-on-iis-75.md]: why is all text bolded from this point forward? Should it be? If not, perhaps the culprit is the three *** before .py?

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-505fd21cb6/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-f0e570a3f3/how-to/). :bow:

**Pull request merged**